ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -125,6 +125,10 @@ def sinh(self, name=None):
     def argmax(self, axis=None, keepdim=False, dtype=None, name=None):
         return ivy.argmax(self._ivy_array, axis=axis, keepdims=keepdim, dtype=dtype)
 
+    @with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
+    def argmin(self, axis=None, keepdim=False, dtype=None, name=None):

```

Comment:
Please implement the exact same signature of [argmin](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/argmin_en.html).

Reply:
made a small mistake i will resolve it into a new PR